event_sourcery/aggregate.py

```diff
@@ -1,10 +1,12 @@
 from contextlib import contextmanager
-from typing import Iterator
+from typing import ClassVar, Iterator
 
 from event_sourcery.interfaces.base_event import Event
 
 
 class Aggregate:
+    category: ClassVar[str]
```

Comment:
I'm not sure about what choice we should make here.
Module path + class name? Convert camel case to snake?

Reply:
Or maybe we can assume by default there is no category cause we want to have one event store per component?

So maybe 
```python
category: ClassVar[str] = ""
```

Reply:
I'd default to the class name I guess. That way it'll be opaque if someone is not using it. An empty category also sounds like an option.

> maybe we can assume by default there is no category cause we want to have one event store per component?
I'm not convinced this will be the default setup. People will be trying to get things work with minimal required effort and having event store per component is not that

Reply:
At this moment I would suggest empty category and let's postpone this decision. But needs to be solved before v1

Reply:
agreed, let's move forward